src/routes/current-studies/Content.svelte

```diff
@@ -52,7 +52,7 @@ function leaveStudy(studyID) {
     <hr />
     <div class="studies">
 
-    {#each studies as study, i (study.addon_id)}
+    {#each studies as study}
```

Comment:
@hamilton I'm not sure if this is the correct fix, but I could not figure out what the `{#each}` block here was doing based on the Svelte docs... we don't seem to be using the loop counter (`i`) and I'm not sure what `(study.addon_id)` does here?

Reply:
Regardless of the fix, I think the test I've added will exercise this :)

Reply:
Yes, we don't need `i` but we will probably need the `(study.addon_id)` key. This tells us that if this value changes for this iterable element, to make sure to update the component. React has a similar concept in the `key` prop.

Reply:
Without this, if the study order changes (for instance once we make studies sortable, or if some other request changes the studies displayed through an async update) the rendered list of components won't update properly.